src/openeo_test_suite/tests/general/test_compliance.py

```diff
@@ -0,0 +1,1494 @@
+from openapi_core import Spec
+import pytest
+import requests
+import openeo_test_suite.tests.general.compliance_util as conformance_util
+import uuid
+
+
+# region Test Fixtures
+
+
+@pytest.fixture(scope="session")
+def base_url(request):
+    return conformance_util.get_base_url(request=request)
+
+
+@pytest.fixture(scope="session")
+def domain(request):
+    return conformance_util.get_domain(request=request)
+
+
+@pytest.fixture(scope="session")
+def spec(request):
+    return conformance_util.adjust_spec_json(
+        conformance_util.get_spec_path_json(),
+        conformance_util.get_base_url(request),
+        conformance_util.get_domain(request),
+    )
+
+
+@pytest.fixture(scope="session")
+def bearer_token(pytestconfig):
+    bearer_token = conformance_util.get_access_token(pytestconfig)
+    return f"Bearer {bearer_token}"
+
+
+# endregion
+
+# region Test Implementations
+
+
+def test_GET_backend_info(base_url: str, spec: Spec, bearer_token: str):
+    """
+    tests all the generic GET endpoints that require neither setup nor cleanup
+
+    setup: None
+    testing: test response by API for GET requests
+    cleanup: None
+
+    """
+    endpoint_path = ""
+    test_name = "Backend Info"
+
+    # Run through all the generic GET endpoints and test their response to a proper request.
+
+    fail_log = conformance_util.test_endpoint(
+        base_url=base_url, endpoint_path=endpoint_path, test_name=test_name, spec=spec
+    )
+
+    assert fail_log == ""
+
+
+def test_GET_well_known(domain: str, spec: Spec, bearer_token: str):
```

Comment:
Does this test take into consideration that the well-known discovery should not be part of the versioned API path?

Reply:
Yes.